data/narratives.yml

```diff
@@ -1,23 +1,58 @@
 management/zone/.*:
-  - 'a#manage.admin-only':
+  - li.manage-tab.active:
     - step: 1
-    - text: 'Welcome! This is the landing page for admins. If at any point you would like to navigate back to this page, click on this tab!'
-  - 'li.facility':
+    - text: Welcome! This is the landing page for admins. If at any point you would like to navigate back to this page, click on this tab!
```

Comment:
Yeah, we can wrap it with gettext here: https://github.com/MCGallaspy/ka-lite/blob/inline/kalite/inline/static/js/inline/views.js#L109